src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  function add(num) {
```

Comment:
```suggestion
  function add(value) {
```
Use meaningful argument name